Dockerfile

```
@@ -1,31 +1,27 @@
-FROM ubuntu:18.04
-
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    docbook-xsl \
-    docbook-xml \
-    fop \
-    libxml2-utils \
-    libsaxon-java \
-    ant \
-    ttf-dejavu \
-    fonts-ipafont \
-    fonts-wqy-microhei \
-    wget \
-    && rm -rf /var/lib/apt/lists/*
-
-RUN wget https://www.xmlmind.com/ditac/_whc/whc-3_1_1.zip -O /tmp/whc.zip \
-    && unzip /tmp/whc.zip -d /opt \
-    && rm /tmp/whc.zip
+FROM adoptopenjdk/openjdk11:alpine-jre
+
+RUN apk add wget docbook-xsl docbook-xml
```

```suggestion
RUN apk add wget docbook-xsl docbook-xml font-noto font-noto-cjk
```

If we can change pdf font to be noto font, a size can be
SIZE 323.8 MiB  BLOB  168.5 MiB

---

I think we can publish TWO container images. One is without font that does not support PDF gen. Another is one with noto font, with tag "pdfgen"